{{cookiecutter.github_repository_name}}/{{cookiecutter.app_name}}/users/serializers.py

```diff
@@ -7,8 +7,8 @@ class UserSerializer(serializers.ModelSerializer):
 
     class Meta:
         model = User
-        fields = ('id', 'username', 'auth_token', 'first_name', 'last_name',)
```

Comment:
@glemmaPaul this removes the quote from `'first_name'` as well.  Do you mind updating it so that that quote is included?